frontend/src/__tests__/cypress/cypress/tests/e2e/pipelines/pipelines.cy.ts

```diff
@@ -0,0 +1,194 @@
+        'base64',
+      ),
+    };
+    cy.fixture('resources/yaml/data_connection.yml').then((yamlContent) => {
+      const modifiedYamlContent = replacePlaceholdersInYaml(
+        yamlContent,
+        dataConnectionReplacements,
+      );
+      const tempFilePath = 'cypress/temp_data_connection.yaml';
+      applyOpenShiftYaml(modifiedYamlContent, tempFilePath).then((result) => {
+        expect(result.code).to.eq(
+          0,
+          `ERROR applying YAML content
+                                      stdout: ${result.stdout}
+                                      stderr: ${result.stderr}`,
+        );
+      });
+    });
+
+    // Configure Pipeline server: Create DSPA Secret
+    const dspaSecretReplacements = {
+      DSPA_SECRET_NAME: dspaSecretName,
+      NAMESPACE: projectName,
+      AWS_ACCESS_KEY_ID: Buffer.from(AWS_PIPELINES_BUCKET.AWS_ACCESS_KEY_ID).toString('base64'),
+      AWS_SECRET_ACCESS_KEY: Buffer.from(AWS_PIPELINES_BUCKET.AWS_SECRET_ACCESS_KEY).toString(
+        'base64',
+      ),
+    };
+    cy.fixture('resources/yaml/dspa_secret.yml').then((yamlContent) => {
+      const modifiedYamlContent = replacePlaceholdersInYaml(yamlContent, dspaSecretReplacements);
+      const tempFilePath = 'cypress/temp_dspa_secret.yaml';
+      applyOpenShiftYaml(modifiedYamlContent, tempFilePath).then((result) => {
+        expect(result.code).to.eq(
+          0,
+          `ERROR applying YAML content
+                                      stdout: ${result.stdout}
+                                      stderr: ${result.stderr}`,
+        );
+      });
+    });
+
+    // Configure Pipeline server: Create DSPA
+    const dspaReplacements = {
+      DSPA_SECRET_NAME: dspaSecretName,
+      NAMESPACE: projectName,
+      AWS_S3_BUCKET: AWS_PIPELINES_BUCKET.BUCKET_NAME,
+    };
+    cy.fixture('resources/yaml/dspa.yml').then((yamlContent) => {
+      const modifiedYamlContent = replacePlaceholdersInYaml(yamlContent, dspaReplacements);
+      const tempFilePath = 'cypress/temp_dspa.yaml';
+      applyOpenShiftYaml(modifiedYamlContent, tempFilePath).then((result) => {
+        expect(result.code).to.eq(
+          0,
+          `ERROR applying YAML content
+                                      stdout: ${result.stdout}
+                                      stderr: ${result.stderr}`,
+        );
+      });
+    });
+  });
+
+  after(() => {
+    // Delete provisioned Project
+    deleteOpenShiftProject(projectName).then((result) => {
+      expect(result.code).to.eq(
+        0,
+        `ERROR deleting ${projectName} Project
+                                    stdout: ${result.stdout}
+                                    stderr: ${result.stderr}`,
+      );
+    });
+  });
+
+  it('An admin User can Import and Run a Pipeline', () => {
+    // Login as an admin
+    cy.visitWithLogin('/', ADMIN_USER);
+    cy.findByRole('banner', { name: 'page masthead' }).contains(ADMIN_USER.USERNAME);
```

Comment:
This assertion is irrelevant to the test case.

Reply:
Fixed in [ea6d831](https://github.com/opendatahub-io/odh-dashboard/pull/3028/commits/ea6d8314940ab4737edffbf374ae294ec55400c1)